LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua

```diff
@@ -78,10 +78,73 @@ local function GetStringFromValue(value)
     return value
 end
 
+local FAQ_ICON_COLOR = ZO_ColorDef:New("FFFFFF") -- white
+local FAQ_ICON_MOUSE_OVER_COLOR = ZO_ColorDef:New("B8B8D3") -- dark-blue/white
+
 local function GetColorForState(disabled)
     return disabled and ZO_DEFAULT_DISABLED_COLOR or ZO_DEFAULT_ENABLED_COLOR
 end
 
+local function CreateFAQTexture(parent, controlData, controlName, anchorPoint, anchorRelativeControl, anchorRelativePoint, anchorX, anchorY)
+    anchorPoint = anchorPoint or TOPLEFT
+    anchorRelativeControl = anchorRelativeControl or parent
+    anchorRelativePoint = anchorRelativePoint or TOPLEFT
+    anchorX = anchorX or 0
+    anchorY = anchorY or 0
+    local onMouseExitAlpha = 0.4
+
+    local helpUrl = controlData and controlData.helpUrl and GetStringFromValue(controlData.helpUrl)
+    if not helpUrl or helpUrl == "" then return end
+    local faqControl = wm:CreateControl(nil, parent, CT_TEXTURE)
+    faqControl:SetHidden(false)
+    faqControl:ClearAnchors()
+    faqControl:SetDrawLayer(DL_OVERLAY)
+    faqControl:SetTexture("\\ESOUI\\ART\\miscellaneous\\help_icon.dds") --small (i): \\ESOUI\\ART\\help\\help_tabicon_tutorial_up.dds
+    if controlData.isHalfWidth then
```

Comment:
Is this really needed? Can't they just have the same size in both cases?

Reply:
I had changed it in size by 2 pixels to make it look and anchor properly for half width controls in submenus. Under some circumstances it was missplaced if the size was 23 pixels as well.
Couldn't rebuild the problem with my current tests thoug.
So I guess 23pixels would be okay for both.